src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing_through.cpp

```diff
@@ -68,6 +68,23 @@ void prepare_primitive_fusing_through::run(program& p) {
         return pass_through;
     };
 
+    auto broadcastable = [&](const ov::PartialShape& in_pshape, const ov::PartialShape& out_pshape) {
+        if (in_pshape.is_dynamic() || out_pshape.is_dynamic())
+            return false;
+
+        if (in_pshape.size() > out_pshape.size())
+            return false;
+
+        size_t min_size = std::min(in_pshape.size(), out_pshape.size());
+
+        for (size_t i = 0; i < min_size; ++i) {
+            if (!(in_pshape[i] == 1 || out_pshape[i] == 1 || in_pshape[i] == out_pshape[i])) {
+                return false;
+            }
+        }
+        return true;
+    };
```

Comment:
As you know "broadcastable" is being used in many places. Could you please add this a common function?

Reply:
done